src/cpp/src/continuous_batching/cache_manager.hpp

```diff
@@ -38,15 +38,19 @@ class CacheManager {
         // extract information about inference device
         ov::CompiledModel compiled_model = request.get_compiled_model();
         std::vector<std::string> execution_devices = compiled_model.get_property(ov::execution_devices);
-        OPENVINO_ASSERT(execution_devices.size() == 1, "Contituous batching: execution device is expected to be CPU or GPU, but got ", execution_devices.size(), " devices");
+        bool all_gpu_device =
+            !execution_devices.empty() &&
```

Comment:
How can `execution_devices` be empty?

Reply:
execution_devices can not be empty, will remove this.